src/lib/index.ts

```diff
@@ -7,6 +8,12 @@ export type PredicateFn = (e: UniDriver, idx?: number, array?: UniDriver[]) => P
 
 export type ReducerFn<T> = (acc: T, curr: UniDriver, idx?: number, array?: UniDriver[]) => T;
 
+export type MouseUniDriver = {
+	move?: (toCoordinates: {x: number, y: number}) => Promise<void>;
```

Comment:
why is this optional?
I see it's implemented by all others

Reply:
just a typo